client/model/saltModel.js

```diff
@@ -40,6 +40,10 @@ define([
             var checkObjContents = function(obj1, obj2) {
                 var ret = true;
                 for (var each in obj1) {
+                    // ignore 'zoom_version' as it is not in zookeeper and therefore only in the salt return
```

Comment:
my bad, but it's not in the path that I am updating with the pillar editor - which is the problem here